<ui version="4.0" >
 <class>QRL_ScopesManager</class>
 <widget class="QDialog" name="QRL_ScopesManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>459</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>459</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Scopes Manager</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qrtailab.qrc" >
    <normaloff>:/icons/scope_icon.xpm</normaloff>:/icons/scope_icon.xpm</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QListWidget" name="scopeListWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="Scope" >
      <attribute name="title" >
       <string>Scope</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QCheckBox" name="showCheckBox" >
         <property name="text" >
          <string>Show/Hide</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QToolBox" name="toolBox" >
         <property name="currentIndex" >
          <number>2</number>
         </property>
         <widget class="QWidget" name="Seite" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>166</width>
            <height>231</height>
           </rect>
          </property>
          <attribute name="label" >
           <string>Display</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4" >
           <item row="0" column="0" colspan="2" >
            <widget class="QLabel" name="maxLabel_4" >
             <property name="text" >
              <string>Divider</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QwtCounter" name="dividerCounter" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="numButtons" >
              <number>1</number>
             </property>
             <property name="basicstep" stdset="0" >
              <double>1.000000000000000</double>
             </property>
             <property name="minValue" >
              <double>1.000000000000000</double>
             </property>
             <property name="maxValue" >
              <double>100000.000000000000000</double>
             </property>
             <property name="stepButton1" >
              <number>1</number>
             </property>
             <property name="stepButton2" >
              <number>10</number>
             </property>
             <property name="stepButton3" >
              <number>100</number>
             </property>
             <property name="value" >
              <double>1.000000000000000</double>
             </property>
             <property name="editable" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="dataPointLabel" >
             <property name="text" >
              <string>Data points</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2" >
            <widget class="QwtCounter" name="dataCounter" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="numButtons" >
              <number>0</number>
             </property>
             <property name="basicstep" stdset="0" >
              <double>1.000000000000000</double>
             </property>
             <property name="minValue" >
              <double>5.000000000000000</double>
             </property>
             <property name="maxValue" >
              <double>100000.000000000000000</double>
             </property>
             <property name="stepButton1" >
              <number>100</number>
             </property>
             <property name="stepButton2" >
              <number>1000</number>
             </property>
             <property name="stepButton3" >
              <number>5000</number>
             </property>
             <property name="value" >
              <double>100.000000000000000</double>
             </property>
             <property name="editable" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="maxLabel_3" >
             <property name="text" >
              <string>Frames</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2" >
            <widget class="QwtCounter" name="rrCounter" >
             <property name="numButtons" >
              <number>1</number>
             </property>
             <property name="basicstep" stdset="0" >
              <double>1.000000000000000</double>
             </property>
             <property name="minValue" >
              <double>0.010000000000000</double>
             </property>
             <property name="maxValue" >
              <double>100.000000000000000</double>
             </property>
             <property name="stepButton1" >
              <number>1</number>
             </property>
             <property name="value" >
              <double>20.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="maxLabel_2" >
             <property name="text" >
              <string>sec/div</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2" >
            <widget class="QComboBox" name="dxComboBox" >
             <property name="editable" >
              <bool>true</bool>
             </property>
             <property name="currentIndex" >
              <number>4</number>
             </property>
             <property name="minimumContentsLength" >
              <number>1</number>
             </property>
             <item>
              <property name="text" >
               <string>0.001</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>0.005</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>0.01</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>0.05</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>0.1</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>0.5</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>1</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0" colspan="3" >
            <widget class="QComboBox" name="displayComboBox" >
             <item>
              <property name="text" >
               <string>Continuous Roling</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Overwrite</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Trigger +to-</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Trigger -to+</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>hold</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0" colspan="3" >
            <widget class="QComboBox" name="directionComboBox" >
             <item>
              <property name="text" >
               <string>right to left</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>left to right</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="0" colspan="3" >
            <widget class="QComboBox" name="optionComboBox" >
             <item>
              <property name="text" >
               <string>Show Grid</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>set BG Color</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>set Grid Color</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>186</width>
            <height>129</height>
           </rect>
          </property>
          <attribute name="label" >
           <string>Trigger</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QCheckBox" name="oneShotCheckBox" >
             <property name="text" >
              <string>One Shot</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QPushButton" name="startTriggerPushButton" >
             <property name="text" >
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Trigger offset</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0" >
            <widget class="QwtCounter" name="triggerCounter" >
             <property name="numButtons" >
              <number>0</number>
             </property>
             <property name="basicstep" stdset="0" >
              <double>0.100000000000000</double>
             </property>
             <property name="minValue" >
              <double>-1000000.000000000000000</double>
             </property>
             <property name="maxValue" >
              <double>1000000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QComboBox" name="traceComboBox" >
             <item>
              <property name="text" >
               <string>Trace 1</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0" colspan="2" >
            <widget class="QPushButton" name="triggerPushButton" >
             <property name="text" >
              <string>Manual Trigger</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>196</width>
            <height>256</height>
           </rect>
          </property>
          <attribute name="label" >
           <string>Saving</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>Filename</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2" >
            <widget class="QLineEdit" name="fileLineEdit" />
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Time [s]</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2" >
            <widget class="QwtCounter" name="timeCounter" >
             <property name="layoutDirection" >
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="numButtons" >
              <number>0</number>
             </property>
             <property name="minValue" >
              <double>0.000000000000000</double>
             </property>
             <property name="maxValue" >
              <double>1000000.000000000000000</double>
             </property>
             <property name="value" >
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2" >
            <widget class="QProgressBar" name="saveProgressBar" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="value" >
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="7" column="0" >
            <widget class="QPushButton" name="savePushButton" >
             <property name="text" >
              <string>Save</string>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
             <property name="autoDefault" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="7" column="1" >
            <widget class="QPushButton" name="stopPushButton" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Trace" >
      <attribute name="title" >
       <string>Trace options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="traceNameLineEdit" />
         </item>
        </layout>
       </item>
       <item rowspan="2" row="1" column="0" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Line Width</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QwtCounter" name="lineWidthCounter" >
            <property name="numButtons" >
             <number>0</number>
            </property>
            <property name="basicstep" stdset="0" >
             <double>1.000000000000000</double>
            </property>
            <property name="minValue" >
             <double>1.000000000000000</double>
            </property>
            <property name="maxValue" >
             <double>50.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QCheckBox" name="showTraceCheckBox" >
         <property name="text" >
          <string>Show/Hide</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="traceColorPushButton" >
         <property name="text" >
          <string>Line Color</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>offset</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QwtCounter" name="offsetCounter" >
            <property name="numButtons" >
             <number>0</number>
            </property>
            <property name="basicstep" stdset="0" >
             <double>0.100000000000000</double>
            </property>
            <property name="minValue" >
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maxValue" >
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>div</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QwtWheel" name="offsetWheel" />
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>dy</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Units/div:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QComboBox" name="dyComboBox" >
            <property name="editable" >
             <bool>true</bool>
            </property>
            <property name="currentIndex" >
             <number>10</number>
            </property>
            <property name="maxVisibleItems" >
             <number>20</number>
            </property>
            <item>
             <property name="text" >
              <string>0.001</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>0.002</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>0.005</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>0.01</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>0.02</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>0.05</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>0.1</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>0.2</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>0.5</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>50</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>100</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>1000</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>show</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" >
           <widget class="QCheckBox" name="zeroAxisCheckBox" >
            <property name="text" >
             <string>Zero Axis</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QCheckBox" name="minCheckBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Min</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QCheckBox" name="averageCheckBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Average</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="labelCheckBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Label</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QCheckBox" name="maxCheckBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Max</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QCheckBox" name="unitCheckBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Units/div</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QCheckBox" name="ppCheckBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>PP</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QCheckBox" name="rmsCheckBox" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>RMS</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close Manager</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
  </customwidget>
  <customwidget>
   <class>QwtWheel</class>
   <extends>QWidget</extends>
   <header>qwt_wheel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qrtailab.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>QRL_ScopesManager</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>304</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel" >
     <x>394</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
